src/SdkCommon/AppAuthentication/Azure.Services.AppAuthentication.IntegrationTests/EndtoEndPositiveTests.cs

```diff
@@ -11,6 +11,10 @@
 using Microsoft.Azure.Services.AppAuthentication.IntegrationTests.Helpers;
 using Microsoft.Azure.Services.AppAuthentication.IntegrationTests.Models;
 using Microsoft.Azure.Services.AppAuthentication.TestCommon;
+#if net472
```

Comment:
The only directive we allow is FullNetFx

Reply:
Per separate thread, we will be doing our own release and signing of our package since we require net472.